jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/PerCaseRuntimeManager.java

```diff
@@ -324,12 +367,14 @@ public boolean validate(Long ksessionId, Long processInstanceId) {
 
         @Override
         public void afterProcessCompleted(ProcessCompletedEvent event) {
+            logger.debug("Removing persistence mapping for kieSessionId {} runtime engine {} manager {} and case id",ksessionId, runtime, managerId, caseId);
             mapper.removeMapping(new EnvironmentAwareProcessInstanceContext(event.getKieRuntime().getEnvironment(), event.getProcessInstance().getId()), managerId);
-            removeLocalRuntime(runtime, event.getProcessInstance().getId());                       
+            removeLocalRuntime(runtime, event.getProcessInstance().getId());
         }
 
         @Override
         public void beforeProcessStarted(ProcessStartedEvent event) {
+            logger.debug("Saving persistence mapping for kieSessionId {} runtime engine {} manager {} and case id",ksessionId, runtime, managerId, caseId);
```

Comment:
Same here, missing placeholder for the last param

Reply:
added